### DISCUSSION TITLE
Wii Remote just not being detected

### DISCUSSION THREAD

**Discussion by WaveSmashies** [KeyPuncher/WiinUPro#179] [Help]
Windows 11, Wii Remote with Motion Plus inside. I installed the included drivers and restarted my PC, but it's just not finding anything when I open the Sync Devices menu. Dolphin finds and uses the controller just fine, I use my Dualsense controller with Bluetooth just fine so I know that's working. I'm not sure what's wrong.

**Comment by WaveSmashies** [comment]
I found a really strange workaround. I mentioned Dolphin finds the wiimote just fine. So I had a brainwave. I just opened Dolphin and connected the controller there, then just opened WiinUSoft. Why did this work? No idea. But I would like the issue to be fixed so I don't need to open Dolphin for no reason each time I wanna use my Wii Remote, NES, or SNES Classic Controllers on PC lol.

**Comment by KeyPuncher** [comment]
The included drivers are not for the Wiimotes, those are for XInput emulation and you should not install the Bluetooth portion of the included (but outdated) SCP drivers. The discovery detection of the Wiimote is for pairing and connecting it via Bluetooth which was written before Windows 11 and has been through its up and downs in success as Microsoft continues to make changes to their Bluetooth implementation. I don't expect it to work with Windows 11 necessarily for the time being. Instead of opening Dolphin you could probably connect your controller via Windows' Bluetooth interface. Or, if Dolphin actually pairs the controller, you should not need to re-pair it each time and simply pressing a button should keep the lights flashing while connected.

**Reply by WaveSmashies** [reply]
I still need to open Dolphin to connect the Wii Remote. I also can't get the wii remote to connect with Windows bluetooth menus, even that old one that used to work for me back in the day.
